index.d.ts

```diff
@@ -0,0 +1,32 @@
+export interface IOptions {
+	/**
+	 * Directory to start from.
+	 *
+	 * @default process.cwd()
+	 */
+	cwd: string;
+	/**
+	 * [Normalize](https://github.com/npm/normalize-package-data#what-normalization-currently-entails) the package data.
+	 *
+	 * @default true
+	 */
+	normalize?: boolean;
+}
+
+export interface IPackage {
```

Comment:
`Package` doesn't feel like the best name. It doesn't return the package itself, but rather the package.json data. Maybe `PackageMetadata`?